internal/cephfs/snapshot.go

```diff
@@ -45,29 +45,16 @@ type cephfsSnapshot struct {
 	ReservedID string
 }
 
-func createSnapshot(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, snapID, volID volumeID) error {
-	args := []string{
-		"fs",
-		"subvolume",
-		"snapshot",
-		"create",
-		volOptions.FsName,
-		string(volID),
-		string(snapID),
-		"--group_name",
-		volOptions.SubvolumeGroup,
-		"-m", volOptions.Monitors,
-		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix + cr.ID,
-		"--keyfile=" + cr.KeyFile,
+func (vo *volumeOptions) createSnapshot(ctx context.Context, snapID, volID volumeID) error {
+	fsa, err := vo.conn.GetFSAdmin()
+	if err != nil {
+		util.ErrorLog(ctx, "could not get FSAdmin, can create a snapshot from %s: %s", vo.FsName, err)
```

Comment:
can this be logged as  `util.ErrorLog(ctx, "could not get FSAdmin  %s", err)`

Reply:
sure!